packages/cli/src/lib/bundler/server.ts

```diff
@@ -132,7 +133,7 @@ DEPRECATION WARNING: React Router Beta is deprecated and support for it will be
   const url = resolveBaseUrl(frontendConfig);
 
   const host =
-    frontendConfig.getOptionalString('app.listen.host') || url.hostname;
+    frontendConfig.getOptionalString('app.listen.host') || url?.hostname;
```

Comment:
This is a change isn't it - now it can be undefined, while before it was gonna be like `localhost` or so - right?